apps/mail/lib/ai-providers.ts

```
@@ -0,0 +1,26 @@
+      return openai(model);
+    case 'google':
+      return google(model);
+    case 'ollama':
+      const ollamaClient = createOllama({
+        baseURL: process.env.OLLAMA_BASE_URL ?? 'http://localhost:11434/api',
+        ...(process.env.OLLAMA_API_KEY && {
+          headers: {
+            'Authorization': `Bearer ${process.env.OLLAMA_API_KEY}`
+          }
+        })
+      });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)